### ISSUE TITLE
Error when unzipping downloaded files using default Windows tool (Solution)

### ISSUE THREAD

**Issue by SimionKreimer** [CMRose3355/TomahaqCompanionProgram#1]
I think this may be due to zipping and unizipping the file.
![tomahaqerror_screenshot](https://user-images.githubusercontent.com/13045601/41928860-9601da68-7944-11e8-94f5-c8888658c84d.png)

**Comment by SimionKreimer** [comment]
Update: I do not get this error when cloning the files from Github.

**Comment by CMRose3355** [comment]
Yes, there is a known issue with downloading the files and unzipping using windows default tool.

There are 2 ways to solve it
1. follow the instructions below
2. Unzip the new version of the program with something like 7-zip and not the default windows one. 

So I tracked down the problem shortly after our call. As you noted the issue occurs when unzipping the file using the default windows option. If you unzip the folder using the "7zip" program you do not get the error. 

The reason for this is that for the two files listed here:

![image](https://user-images.githubusercontent.com/9395501/42416649-f5d51138-8229-11e8-8be7-3d0a2741b70d.png)

You need to right click and select "Properties", then at the bottom of this dialog box you need to click "Unblock". This should fix the issue. 

If you use "7zip" this isn't necessary. 

![image](https://user-images.githubusercontent.com/9395501/42416638-c316861e-8229-11e8-94f6-57d3db380d7b.png)


Hopefully this will work.
